src/main/java/net/pms/formats/BMS.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * PS3 Media Server, for streaming any medias to your PS3.
```

Comment:
We can probably use a Universal Media Server comment instead of the old PS3 Media Server one in our new files :)